src/api-client/galileo-client.ts

```diff
@@ -478,6 +478,8 @@ export class GalileoApiClient extends BaseClient {
     instructions?: string,
     chainPollTemplate?: ChainPollTemplate,
     userPrompt?: string,
+    nodeLevel?: string,
```

Comment:
should use an enum or something?  (e.g. StepType)?

Reply:
https://github.com/rungalileo/galileo-js/blob/1b125b8b9f672c1d8b09006edf47b2dbce066baf/src/types/logging/step.types.ts#L39-L46